Makefile

```diff
@@ -43,22 +43,26 @@ controller-gen: ## Download controller-gen locally if necessary.
 ## The workaround is:
 ##   1. move module's go.mod file to a temporary file
 ##   2  copy main go.mod file to a temporary file to preserve changes
-##   3. run "go mod tidy" to ensure main go.mod file contains all required dependecnies
+##   3. run "go mod tidy" to ensure main go.mod file contains all required dependencies
 ##   4. run controller-gen command
 ##   5. restore go.mod files
 
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	mv ./apis/config/go.mod ./apis/config/go_temp.mod
+	mv ./apis/hoh_heartbeat/go.mod ./apis/hoh_heartbeat/go_temp.mod
```

Comment:
how about using a loop:

`for API in ./apis/*; do mv ./apis/$API/go.mod ./apis/$API/go_temp.mod`?

Reply:
done